test/test_lowlevel.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+from pudb.lowlevel import detect_encoding, decode_lines
+from pudb.py3compat import PY3
+
+
+def test_detect_encoding_nocookie():
+    lines = ['Test Проверка']
```

Comment:
Not sure, but I don't think so. On py3 they would be unicodes, on py2 strings. As I can see, `detect_encoding` works with native string format (unicode for 3, str for 2). So it would when receive unicode lines on 2 and bytes on 3.